### ISSUE TITLE
Focus outline for links in 'Ways to help' box makes links difficult to read.

### ISSUE THREAD

**Issue by NickColley** [tatianamac/selfdefined#349]
When tabbing into the 'Ways to help' box the contrast between the [focus outline and the text is too low at 2.57](https://contrast-ratio.com/#%23212121-on-%23ad3448
).

![Image of Ways to help box, shows a link highlighted with poor contrast](https://user-images.githubusercontent.com/2445413/95382027-eace0980-08e0-11eb-9f29-f8bf340df24e.png)

If we were to use the background colour of the box for the text colour the [contrast would be improved to at least 5.24](https://contrast-ratio.com/#%23ebebeb-on-%23ad3448
).

I would argue to go for a [brighter white to ensure a better contrast at 6+](https://contrast-ratio.com/#white-on-%23ad3448)

One way to accomplish might be to change the `.box a` selector to `.box a:not(:focus)`.

I've seen you're using CSS Custom Properties, so that might be another nice way to resolve this. For example by introducing a `--focus-text-color` variable that can be assigned on the `.box`.

**Comment by tatianamac** [comment]
@nickcolley  Thanks for submitting this issue. 🙏🏽

@ovlb Is this something that got overlooked into the light-dark mode PR? I felt like we went back and forth in the PR re: this very topic.

**Comment by ovlb** [comment]
Indeed, we went back and forth on this and @nickcolley sees there is not what is specified. The Firefox Accessibility Dev Tools report as follows:

<img width="234" alt="Screen Shot of the contrast ratio of a hovered/focused link inside the box. The ratio is reported with 10.47. being WAI AAA compliant" src="https://user-images.githubusercontent.com/7645967/95424213-9bb8c080-0942-11eb-9d71-ebfa9b82a626.png">

@nickcolley Can you tell me which browser and OS/monitor you use? 

Thanks,
Oscar

**Comment by NickColley** [comment]
I was on Linux, with Google Chrome, using Dark mode. It may be some weirdness with CSS Custom Properties perhaps. Hmm.

I believe there are similar issues with the light theme, they're actually much more severe.

<img width="237" alt="" src="https://user-images.githubusercontent.com/2445413/95437159-906a9280-094c-11eb-90e5-84497cb9146f.png">

Update: Seeing the issues on OSX Chrome and Firefox.

**Comment by ovlb** [comment]
@nickcolley Hmm. `--clr-accent` resolves to `pink`, but [the values browsers implement are specced](https://www.w3.org/TR/2018/REC-css-color-3-20180619/#svg-color). Can you have a look into your browser dev tools and post to which values text and background resolve in the end? 

Re light mode: Seeing this too. This slipped through our QA. 🙈  I’d say we need a darker shade of `pink` for the black bg/white text box.
